cheroot/server.py

```diff
@@ -1866,8 +1900,39 @@ def prepare(self):  # noqa: C901  # FIXME
         self.ready = True
         self._start_time = time.time()
 
+    def _serve_unservicable(self):
+        """Serve connections we can't handle a 503."""
+        while self.ready:
+            try:
+                conn = self._unservicable_conns.get()
+            except QueueShutDown:
+                return
+            request = HTTPRequest(self, conn)
+            try:
+                request.simple_response('503 Service Unavailable')
+            except (socket.error, errors.FatalSSLAlert):
```

Comment:
`socket.error` is an ancient alias for `OSError`
```suggestion
            except (OSError, errors.FatalSSLAlert):
```

Reply:
I don't blame you :) There's also a lot of ancient compat hacks all over. I think I was cleaning out some Python 2.4 compat things in CherryPy the other day…